.github/workflows/publish-docs.yaml

```diff
@@ -10,6 +10,11 @@ jobs:
     if: ${{ !github.event.release.prerelease }}
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout
+        uses: actions/checkout@v2      
+        with:
+          token: ${{ secrets.WEAVEWORKSBOT_TOKEN }}
```

Comment:
This isn't required.

Reply:
Which on? The token? Oh, that's probably right. :)

Reply:
Yeah, the token :)